### DISCUSSION TITLE
`digdeeper` / `spyware.neocities.org`

### DISCUSSION THREAD

**Discussion by smnthermes** [FiltersHeroes/KADhosts#60] [Q&A] [answered]
digdeeper.neocities.org and spyware.neocities.org aren't fake news websites:
https://github.com/PolishFiltersTeam/KADhosts/blob/master/KADfakeHosts.txt

**Comment by hawkeye116477** [comment]
It was already discussed and they are talking about things which they aren't fully understand and extrapolating a lot of things. 
For example, I can't agree that self updating is spyware. They are a bit too paranoid. 
It's best to cut the cable from the Internet and it will be done :smiley:

Anyway, it's duplicate of https://github.com/PolishFiltersTeam/KADhosts/issues/26.

**Comment by hawkeye116477** [comment]
> Self updates are a spyware feature since they are usually ways for the developer of a program to put spyware into their software without presenting it in a prominent way where the user can understand what they are giving up when they download the update.

These are great theories... When I leave the house, a tree will fall and I'll broke the leg...
Similar arguments as with anti-vaccine activists.

------------

If software is open-source then you could look at it's code, analyze, check it, you mustn't guessing. In my opinion, you are abusing „spyware" word.

We'll think up again about this. However, KADfake Hosts contains also some controversial websites.

**Comment by krystian3w** [comment]
Or change type of block from broken page to only uBlock0 / AdGuard warming:

```adb
||digdeeper.neocities.org^$document
||spyware.neocities.org^$document
```

**Reply by krystian3w** [reply, marked as answer]
List do not longer block  controversial websites.

**Comment by ghost** [comment]
As the other spyware website maintainer, I approve the message above
